apps/builder/app/builder/features/pages/pages.tsx

```diff
@@ -392,7 +392,7 @@ const PagesTree = ({
               }}
             >
               <TreeNode
-                level={item.level}
+                level={item.level - 1}
```

Comment:
<img width="186" alt="image" src="https://github.com/user-attachments/assets/98a7816a-da9a-493e-ac4e-085a4ae0b17e" />

Reply:
how did you repro? I tested didn't see dnd being broken

Reply:
what am I looking at on the screenshot?

Reply:
Just a folder with nested page and another folder. Expanded.
1. arrow requires space, that one level
2. dnd now shows random indicator

Just revert this change

Reply:
Now I get it, spacing on the left looked weird